src/components/molecules/m-footer-middle.vue

```diff
@@ -20,7 +20,7 @@
         <atom-text class="is-size-6 mb-2" :text="text2" />
       </div>
       <div class="column">
-        <a href="https://www.instagram.com/parkspot/?hl=en" target="_blank">
+        <a href="https://www.instagram.com/parkspot/" target="_blank">
```

Comment:
please redirect to our insta page: https://www.instagram.com/parkspot.in/

Reply:
done